web/app/js/components/EmptyCard.jsx

```diff
@@ -12,24 +12,16 @@ const styles = () => ({
   },
 });
 
-const EmptyCard = ({ content, classes }) => {
+const EmptyCard = ({ classes }) => {
   return (
     <Card className={classes.card} elevation={3}>
       <CardContent>
         <Typography>
-          {content}
+          <Trans>No data to display</Trans>
```

Comment:
This is an example of translating a static text string.

Also, we were never passing in a prop `content` to this component, so we could remove the propTypes checks.